devenv/lib/config.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
```

Comment:
unittest coverage for this will be addressed in https://github.com/getsentry/devenv/pull/80